src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store/Customizations/DataLakeStoreCustomizationHelper.cs

```diff
@@ -18,7 +18,7 @@ internal static class DataLakeStoreCustomizationHelper
         /// This constant is used as the default package version to place in the user agent.
         /// It should mirror the package version in the project.json file.
         /// </summary>
-        internal const string PackageVersion = "1.0.2-preview";
```

Comment:
I have removed the AutoRest versioning in the license header from all the custom classes, since they are not auto-genned.